app/models/alert_types/wiki_expert_notification_alert.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: alerts
+#
+#  id             :integer          not null, primary key
+#  course_id      :integer
+#  user_id        :integer
+#  article_id     :integer
+#  revision_id    :integer
+#  type           :string(255)
+#  email_sent_at  :datetime
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  message        :text(65535)
+#  target_user_id :integer
+#  subject_id     :integer
+#  resolved       :boolean          default(FALSE)
+#  details        :text(65535)
+
+class WikiExpertNotificationAlert < Alert
+  def wiki_experts_email
+    CoursesUsers.where(course:, user: SpecialUsers.wikipedia_experts,
+                       role: CoursesUsers::Roles::WIKI_ED_STAFF_ROLE)
+                .includes(:user).pluck('users.email')
+  end
+
+  def send_email
+    return if emails_disabled?
+
+    WikiExpertNotificationMailer.send_email(self)
+    update(email_sent_at: Time.zone.now)
+  end
+
+  def url
+    course_url
+  end
+
+  def main_subject
+    "Alert: A Student Joined Early in Course #{course.title}"
```

Comment:
I think "EarlyEnrollmentAlert: #{course.slug}" would be good for the subject line of the email.